docs/latest/design/system-design.md

```diff
@@ -39,7 +39,56 @@ defined as Kubernetes resources that provide the following services:
 * Infrastructure Management- Manage the data plane infrastructure, i.e. deploy, upgrade, etc. This configuration is
   expressed through [GatewayClass][gc] and [Gateway][gw] resources. The `EnvoyProxy` [Custom Resource][cr] can be
   referenced by `gatewayclass.spec.parametersRef` to modify data plane infrastructure default parameters,
-  e.g. expose Envoy network endpoints using a NodePort service instead of a LoadBalancer service.
+  e.g. expose Envoy network endpoints using a `ClusterIP` service instead of a `LoadBalancer` service, but doesn't support
```

Comment:
Just my two cents: The desgin doc is just a high-level description of EG, so we may want to move this yaml snippet to user doc or just remove it.

Reply:
@Xunzhuo  did this at #1203, maybe I just need to change the description.